modules/govuk/manifests/apps/mapit.pp

```diff
@@ -56,6 +56,9 @@
       ensure => present,
     }
 
+    # Install postgrest developments dependencies
```

Comment:
At least for me, I don't see this adding any information over what the line of code below says. Maybe put something like "Ensure the libpq-dev package is installed", as that gives some more specific information on what the code should do?